code/api/src/modules/order/types.js

```diff
@@ -14,12 +15,16 @@ const OrderType = new GraphQLObjectType({
     id: {type: GraphQLInt },
     user: { type: UserType },
     crate: { type: CrateType },
-    productDeliveries: { type: new GraphQLList( ProductDeliveryType )},
+<<<<<<< HEAD
```

Comment:
Looks like there is a conflict from a rebase. Probably need to fix this before merge! @jdcarey128 @GarrettCottrell @the-color-bliu